### ISSUE TITLE
tensorflow- After hundreds of epochs, my total_loss stay around 0.6~0.7, and not decreased

### ISSUE THREAD

**Issue by turboLIU** [DeepScale/SqueezeNet#49]
I wrote my own network,  when start training my network,  my total_loss decreased to 0.6~0.7  and does not goingdown any more everytime.   I try to finetuning bias and stddev, but nothing worked.    
 And  What is  more,  the accuracy does not increase,  it is like some random number...
what can i do to fix the network?

**Comment by turboLIU** [comment]
somebody help me!!!!!!

**Comment by Liberus** [comment]
Hello!

Please use appropriate tech discussion platform for your question. If you wrote your own network then it has nothing to do with SqueezeNet. For example, you can use one of the stackexchange sites.

Best regards, Andrii
